core/arch/arm/include/sm/optee_smc.h

```diff
@@ -317,6 +322,9 @@
 /* Secure world supports pre-allocating RPC arg struct */
 #define OPTEE_SMC_SEC_CAP_RPC_ARG		BIT(6)
 
+/* Secure world provisions thread for system service invocation */
+#define OPTEE_SMC_SEC_CAP_SYSTEM_THREAD		BIT(8)
```

Comment:
What happened with `BIT(7)` and why is this separated from the other "SEC_CAPs"?

Reply:
indeed. sorry, a leftover  from https://github.com/OP-TEE/optee_os/pull/5793 that adds another funcID.